### PR TITLE
[13.0] [MIG] website_sale_product_style_badge: Migration to v13

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT26957

please @Tardo @pedrobaeza review this

### CODE REVIEW

File: website_sale_product_style_badge/__manifest__.py [hunk 1]
```diff
@@ -2,13 +2,13 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "Website Sale Product Style Custom Badge",
-    "version": "12.0.1.0.0",
+    "version": "13.0.1.0.0",
     "category": "Website",
     "author": "Tecnativa, " "Odoo Community Association (OCA)",
```
Comment (chienandalu) [inline website_sale_product_style_badge/__manifest__.py]:
```suggestion
    "author": "Tecnativa, Odoo Community Association (OCA)",
```

**Review by chienandalu — APPROVED** [review]
Just a tiny fix

**Comment by CarlosRoca13** [comment]
@chienandalu change done

**Review by chienandalu — APPROVED** [review]
:+1:

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**